### DISCUSSION TITLE
Pico Motion Tracker support for ALVR?

### DISCUSSION THREAD

**Discussion by Originalas** [alvr-org/ALVR#1825] [General]
Can Pico Motion support be added to ALVR?  

https://developer-global.pico-interactive.com/document/unity/body-tracking/

**Comment by zmerp** [comment]
We can add support when they add and OpenXR extension.

**Reply by 755713372** [reply]
In December, 2023, the beta version was released to support tracking, but it entered the stream after starting the tracking calibration after starting the tool.
